inc/VIPRCalculator.h.bsk

```diff
@@ -0,0 +1,62 @@
+#ifndef VIPRCALCULATOR_H
```

Comment:
What file is `.h.bsk`?

Reply:
It was a typo, the file should have been deleted with all other backup files.